### PR TITLE
[2.0] Remove deprecated asShape / adapter classes

### PR DESCRIPTION
**Pull request by jorisvandenbossche**

### CODE REVIEW

**Comment by jorisvandenbossche** [comment]
Updated / fixed conflicts now the other PR is merged.

@sgillies @mwtoews even more lines to remove here ;)

**Comment by sgillies** [comment]
It's a shame that code coverage goes down because this dubious feature was well tested :)

File: shapely/geometry/geo.py [hunk 1]
```diff
@@ -5,12 +5,12 @@
 
 from shapely.errors import ShapelyDeprecationWarning
 
-from .point import Point, asPoint
-from .linestring import LineString, asLineString
-from .polygon import Polygon, asPolygon
-from .multipoint import MultiPoint, asMultiPoint
-from .multilinestring import MultiLineString, asMultiLineString
-from .multipolygon import MultiPolygon, MultiPolygonAdapter
```
Comment (sgillies) [inline shapely/geometry/geo.py:13]: :hocho: :wave:

**Comment by jorisvandenbossche** [comment]
OK, thanks for the review, then I will do my first merge ;)

**Comment by jorisvandenbossche** [comment]
@sgillies hmm, it seems that although I am now a member of the Toblerity organization, I don't yet have "write access" to the Shapely repository. So that's maybe yet another setting?

**Comment by sgillies** [comment]
@jorisvandenbossche ah, right, I did miss the Shapely settings, but you have write perms now.

**Comment by jorisvandenbossche** [comment]
Great, thanks!

**Pull request merged**